src/ApiStatus.ts

```diff
@@ -92,16 +93,9 @@ export default class ApiStatus {
 
     private getLastUpdate(): string {
         const timeDiff = Date.now() - this.lastCheckTime;
-        const min = Math.floor(timeDiff / 1000 / 60);
-        if (min >= 1) {
-            const minutes = min == 1 ? "minute" : "minutes";
-            return "Last refresh: " + min + " " + minutes +" ago";
-        } else {
-            const sec = Math.floor((timeDiff - min * 60000) / 1000);
-            const seconds = sec == 1 ? "second" : "seconds";
-            return "Last refresh: " + sec + " " + seconds + " ago";
-        }
+        return "Last refresh: " + prettyMs(Math.max(timeDiff, 1000), { verbose: true, secDecimalDigits: 0 }) + " ago";
```

Comment:
Is this correct? Specifically the part where it cant be lower than 1 second.

Reply:
I did that on purpose. I wanted to avoid showing `5 milliseconds ago` to the user. That adds as much information as 1 second ago. 
The update timer is also not entirely correct. It would be better if your api status api returns a 'lastUpdateTime' field and we use that to show the duration since the last update.